src/pykeen/nn/message_passing.py

```diff
@@ -757,14 +749,12 @@ def _real_forward(self) -> torch.FloatTensor:
 
         return x
 
-    def forward(
+    def _real_forward(
```

Comment:
I'd prefer a better name than this, otherwise lgtm

Reply:
any suggestion? :sweat_smile:

Reply:
`lookup()`? I just imagine if an external dev needs to implement their own representation, needing to override a hidden method might be confusing. Otherwise, just call this `_forward` or `_forward_helper` if you want to keep it  private

Reply:
5355cbf